### DISCUSSION TITLE
implements i18n

### DISCUSSION THREAD

**Discussion by G0S73** [EDUJOS/tweet-cli-v2#44] [Ideas]
It would be nice if the first time the application is started a customization menu appears to the user, that is, being able to configure the tweet-cli language and among other things.

**Comment by EDUJOS** [comment]
It's great that you thought about it but in fact we are already working on it!

[Implement multi languages ​​with i18n Release Required #23](https://github.com/EDUJOS/tweet-cli-v2/issues/23)
